paasta_tools/chronos_tools.py

```diff
@@ -300,6 +300,17 @@ def check_parents(self):
                                % ", ".join(badly_formatted))
         return True, ''
 
+    def check_cmd(self):
+        command = self.get_cmd()
+        if command:
+            try:
+                parse_time_variables(command)
+                return True, ""
+            except (ValueError, KeyError, TypeError):
+                return False, "Unparseable command. Hint: do you need to escape % chars?"
```

Comment:
Can you go ahead and spit out what the command was to the user that was unparseable?